ComponentKit/Utilities/CKComponentActionInternal.h

```diff
@@ -57,11 +61,11 @@ class CKTypedComponentActionBase {
 
   bool operator==(const CKTypedComponentActionBase& rhs) const;
 
-private:
   // Destroying this field calls objc_destroyWeak. Since this is the only field
   // that runs code on destruction, making this field the first field of this
   // object saves an offset calculation instruction in the destructor.
   __weak id _targetOrScopeHandle;
+  dispatch_block_t _block;
```

Comment:
I'm thinking that may be what I do if this has a big impact on binary size. We'd switch to an Obj-C wrapper for these params that's constructed with a C function. Then as we pass around this struct it really just becomes a single strong pointer in the actual copy constructors.